pkgs/tools/networking/openssh/default.nix

```diff
@@ -13,16 +13,16 @@ let
   gssapiPatch = fetchpatch {
     name = "openssh-gssapi.patch";
     url = "https://salsa.debian.org/ssh-team/openssh/raw/"
-      + "d80ebbf028196b2478beebf5a290b97f35e1eed9"
+      + "f5e0907876b1168f8d77fb4a8b96fc62965dd6f4"
       + "/debian/patches/gssapi.patch";
-    sha256 = "14j9cabb3gkhkjc641zbiv29mbvsmgsvis3fbj8ywsd21zc7m2wv";
+    sha256 = "1cnqlkrxzq7f2yy1iw9x0cp1w8g1skdc7q8r5x7s7chg3i0x5w92";
   };
 
 in
 with stdenv.lib;
 stdenv.mkDerivation rec {
-  pname = "openssh";
-  version = if hpnSupport then "7.8p1" else "7.9p1";
+  name = "openssh-${version}";
```

Comment:
What is the reason of moving away from ```pname```?

Reply:
Whoops. That was a rebase mistake, I'll fix that.